sdk/devcenter/azure-developer-devcenter/tests/testcase.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import functools
+from devtools_testutils import AzureRecordedTestCase, PowerShellPreparer
+from azure.developer.devcenter import DevCenterClient
+
+
+class DevcenterTest(AzureRecordedTestCase):
+    def create_client(self, tenant_id, dev_center):
+        credential = self.get_credential(DevCenterClient)
+        return DevCenterClient(
+            dev_center=dev_center,
+            tenant_id=tenant_id,
+            credential=credential
+        )
+
+
+DevcenterPowerShellPreparer = functools.partial(
+    PowerShellPreparer, "devcenter", dev_center=os.environ.get("DEVCENTER_NAME", "sdk-default-dc"), tenant_id=os.environ.get("AZURE_TENANT_ID", "88888888-8888-8888-8888-888888888888")
```

Comment:
please create a `conftest.py` like [here](https://github.com/Azure/azure-sdk-for-python/blob/3865d37a157d82f646bb3f2357c2d03257c987df/sdk/loadtestservice/azure-developer-loadtesting/tests/conftest.py). It is very useful in pytest.

Reply:
We have a `conftest.py` file [here](https://github.com/Azure/azure-sdk-for-python/pull/26696/files#diff-4df4081e4b9c4a11c50368ae7f60e37007f9a9ea1c9ca45df35f77d26cad4c37) -- is it not correct?